client/state/imports/uploads/reducer.js

```diff
@@ -1,15 +1,15 @@
 /**
  * Internal dependencies
  */
-import { combineReducers, withoutPersistence } from 'calypso/state/utils';
+import { combineReducers } from 'calypso/state/utils';
 import {
 	IMPORTS_UPLOAD_SET_PROGRESS,
 	IMPORTS_UPLOAD_COMPLETED,
 	IMPORTS_UPLOAD_FAILED,
 	IMPORTS_UPLOAD_START,
 } from 'calypso/state/action-types';
 
-const inProgress = withoutPersistence( ( state = false, action ) => {
+const inProgress = ( state = false, action ) => {
```

Comment:
What's the reason for enabling persistence in those?

Reply:
Reducers that are combined with `combineReducers` get wrapped with `withoutPersistence` [by default](https://github.com/Automattic/wp-calypso/blob/e97f017640bc1311551b6e1f7f0fd2b4a239c986/client/state/utils/reducer-utils.js#L250), this extra call is redundant. In other words, "no persistence" is the default and you need to enable it explicitly.

Most of our `withoutPeristence` wrappers are a result of one of @sgomes' earlier codemods and are not really needed 🙂

Reply:
Mea culpa. The `withoutPersistence` wrapping was done mostly out of fear, but looking back, the only things that would have likely broken without it would have been some reducer tests.

Reply:
I thought that was the case. Thanks, both, for confirming 😉